Documentation/proposals/202304-silences.md

```diff
@@ -0,0 +1,108 @@
+# Silence CRD
+
+* Owners:
+  * [mcbenjemaa](https://github.com/mcbenjemaa)
+* Related Tickets:
+  * [#5452](https://github.com/prometheus-operator/prometheus-operator/issues/5452)
+  * [#2398](https://github.com/prometheus-operator/prometheus-operator/issues/2398)
+* Other docs:
+  * n/a
+
+This document describes the creation of `Silence` Custom Resource Definition that defines Alertmanager silences
+configurations in the Kubernetes way.
+
+# Why
```

Comment:
I think we are missing a few reasons why we think this is useful:
1 - Multi tenancy: Alertmanager cannot be provided in read-only mode but silences CR can be using kubernetes rbac 
2 - Moving the giantswarm silence operator to prometheus operator would provide better community support than what we can currently offer

Reply:
I added them in the goals section

Reply:
I think also we could mention that this can offer a better user experience so they can use the same tools to manage their monitoring pipeline (kyverno for instance, cr validations) whereas if silences are created via jobs or any other mechanism lire direct access to alertmanager, you have to redo this tooling again

Reply:
I have added it, in the goals section.
Please take a look